packages/devtools_app/lib/src/shared/charts/chart_trace.dart

```diff
@@ -180,10 +180,8 @@ class Trace {
       controller.yMaxValue = datum.y;
     }
 
-    final traceIndex = controller.traceIndex(this);
-
     // New data has arrived notify listeners this data needs to be plotted.
-    controller.traceChanged.value = TraceNotifier(traceIndex, data.length - 1);
+    controller.traceChanged.value = TraceNotifier();
```

Comment:
FYI @polina-c this now appears to be a strange pattern we are using a ValueNotifier for. Seems like maybe we should be using a stream instead? This is used to notify the memory chart that we need to recompute. We should file a bug to clean up.

Reply:
This is very old code that needs cleanup. If/when somebody needs to change it, they will do it. Not sure bug will make change. Added TODO.